packages/manager/modules/iam/src/identities/identities.styles.scss

```diff
@@ -0,0 +1,7 @@
+.iam-identities {
+  .iam-identities__field {
+    .oui-field {
+      margin-bottom: 0.5rem !important;
```

Comment:
can't be managed with bootstrap ? or at least with inheritance bootstrap css ?

Reply:
Unfortunately it can't as we use the component version of uikit witch does not expose a property for this class